src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -138,8 +138,8 @@ protected function _aliasConnections()
                 $map['test_' . $connection] = $connection;
             }
         }
-        foreach ($map as $alias => $connection) {
-            ConnectionManager::alias($connection, $alias);
+        foreach ($map as $testConnection => $normal) {
+            ConnectionManager::alias($testConnection, $normal);
```

Comment:
I pushed up another change to correct the parameters of the alias() method too. The code isn't exactly the same as before as in 1162121f1e8b8d0f1df0fb65ddf8ace5af8693d0, the keys were live connection name, if the `test_` was encountered, and in the else case, the `test_` prefix connection name was the key.  Now the keys are always the `test_` connection/alias.